src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,45 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+    private static final int TOTAL = -1;
+    private static final int NUMBERS_COUNT = 12;
+    private static final int FULL_NUMBER = 11;
```

Comment:
This constant is related to NUMBERS_COUNT . Changing only one leads to an error (inconsistency).